### PR TITLE
Added the footer to the main page

### LINKED ISSUE #335: Add a footer to the main page
**We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more.**

Hi, this is a first-timers-only issue. This means we've worked to make it more legible to folks who either haven't contributed to our codebase before, or even folks who haven't contributed to open source before.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

If you have contributed before, consider leaving this one for someone new, and looking through our general help wanted issues. Thanks!

## The Problem:
We need to add a footer to the demo and provide some generic information in it, exact content of the footer is yet to be decided.

## How to Solve?
We need to add a footer to the main index.html file at the following location
https://github.com/publiclab/image-sequencer/blob/88364decf6a6111a0be31fd98010576d5c6f918e/examples/index.html#L88
It should look something like this
```js
<footer class="page-footer font-small">
 </footer>
```
Instance Count 1

### PR DESCRIPTION
**Pull request by Harshithpabbati**
Fixes #335

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Harshithpabbati** [comment]
@publiclab/reviewers
* @publiclab/is-reviewers

**Comment by jywarren** [comment]
Hi, do you think you could share a screenshot? Thanks!

**Comment by Harshithpabbati** [comment]
screenshot from 2019-02-15 12-43-47

**Comment by HarshKhandeparkar** [comment]
@Harshithpabbati you don't need to tag publiclab/reviewers, @publiclab/is-reviewers is enough. The reviewers group includes all the reviewers and also the reviewers of other projects like plots. Thanks!

**Comment by HarshKhandeparkar** [comment]
Also you can remove square brackets around the issue number so that becomes a proper link(like i just did)

**Comment by HarshKhandeparkar** [comment]
Fo you think centered text for the columns will look better?

File: examples/index.html [hunk 1]
```diff
@@ -189,6 +189,26 @@ <h2 style="margin-top:20px">Save</h2>
...
+      <div class="col-md-6">
+        <h2>Need Help?</h2>
+        <p>
+          Post a link to this and ask for help from other community members on <a href="https://publiclab.org/">PublicLab.org</a>
+        </p>
+        <p>
+          <a class="btn btn-primary" href="https://publiclab.org/questions">Ask a question &raquo;</a>
+        </p>
+      </div>
+      <div class="col-md-6">
```
Comment (HarshKhandeparkar) [inline examples/index.html]: A class of col-sm-12 along with col-md-6 will be better

File: examples/index.html [hunk 2]
```diff
@@ -189,6 +189,26 @@ <h2 style="margin-top:20px">Save</h2>
...
   <footer>
     <hr style="margin:20px;"><center><a class="color:grey;" id="clear-cache">Clear offline cache</a></center>
+    <div class="row">
+      <div class="col-md-6">
+        <h2>Need Help?</h2>
+        <p>
+          Post a link to this and ask for help from other community members on <a href="https://publiclab.org/">PublicLab.org</a>
+        </p>
+        <p>
+          <a class="btn btn-primary" href="https://publiclab.org/questions">Ask a question &raquo;</a>
```
Comment (HarshKhandeparkar) [inline examples/index.html]: I don't think linking to publiclab.org is good. Maybe link to GitHub issues? Or swap the ask a question section for something else?

File: examples/index.html (hunk 1 above)
Comment (Harshithpabbati) [reply examples/index.html]: Sure i will make the changes,😁

Comment (Harshithpabbati) [reply examples/index.html]: Shall i change the link to github issues??

File: examples/index.html (hunk 2 above)
Comment (HarshKhandeparkar) [reply examples/index.html]: What is your idea about this? Is github issues good? Or should it link to something else?

**Comment by HarshKhandeparkar** [comment]
You can put an 'x' inside the checkboxes in the pr body(without leaving any spaces) or you can directly change them from the browser as they are interactive. Thanks.

**Comment by HarshKhandeparkar** [comment]
@jywarren nodejs is throwing an error in the tests. Maybe an npm audit is required? This is happening probably because of depandabot version bumps.

**Comment by Harshithpabbati** [comment]
screenshot from 2019-02-15 14-44-35
@HarshKhandeparkar is this ok changed it to issues??

**Comment by HarshKhandeparkar** [comment]
What I meant was that people can ask a question by opening an issue

**Comment by jywarren** [comment]
Oops did this close by mistake?

I think the public lab link is ok for end users, because not everyone knows
how to open a GitHub issue.

**Comment by Harshithpabbati** [comment]
I sent a new pr with link of the github issues.
 https://github.com/publiclab/image-sequencer/pull/776
Check this.

**Pull request closed without merging**